base/compiler/typeinfer.jl

```diff
@@ -66,6 +69,28 @@ Timing(mi_info, start_time) = Timing(mi_info, start_time, start_time, UInt64(0),
 
 _time_ns() = ccall(:jl_hrtime, UInt64, ())  # Re-implemented here because Base not yet available.
 
+"""
+    Core.Compiler.Timings.clear_and_fetch_timings()
+
+Return, then clear, the previously recorded type inference timings (`Core.Compiler._timings`).
```

Comment:
```suggestion
Return, then clear, the previously recorded type inference timings.
```
This was not correct anymore, and also we don't want people to be accessing the correct vector (`_finished_timings`) by name anyway.